scripts/browse

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# View a data directory in JBrowse
+main() {
+    if [[ "$1" == "--build" ]]; then
+	1>&2 echo "Building browser image jb2:latest"
+	docker build -t jb2 docker/jbrowse.dockerfile
```

Comment:
When trying `SWG_DEFAULT_USER=1 scripts/browse --build`, I am getting this error: 
```
Building browser image jb2:latest
[+] Building 0.0s (0/0)                                                                                               docker:desktop-linux
ERROR: unable to prepare context: path "docker/jbrowse.dockerfile" not found
```
However, the build works if I change L7 to:
`docker build -t jb2 -f docker/jbrowse.dockerfile .`

I don't know if this is another of case macOS being macOS? 😄

Reply:
Thanks for spotting this, fixed in e438d02 : I had omitted a dot.

For this build, ``SWG_DEFAULT_USER=1` should not be necessary.